tox.ini

```diff
@@ -92,8 +92,10 @@ commands =
 
 [flake8]
 ignore =
-	E501 # line too long (86 > 79 characters)
-	W503 # line break before binary operator
+	# line too long (86 > 79 characters)
```

Comment:
Ha ha, I had to do the same thing in Rdiffweb, Did you figure out why ? I guess it's related to a recent version of flake8..

Reply:
Yes, definitely: https://github.com/PyCQA/flake8/issues/1760